src/web/client/common/remoteFetchProvider.ts

```diff
@@ -158,7 +158,7 @@ async function createContentFiles(
         PowerPlatformExtensionContextManager.updateSingleFileUrisInContext(vscode.Uri.parse(fileUri));
 
         // Display only the last file
-        vscode.window.showTextDocument(vscode.Uri.parse(fileUri));
+        vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(fileUri), {background: true, preview: false});
```

Comment:
this will still prompt to open the default file in editor right?

Reply:
Prompt?
But yes, it will open the default file in editor just as it used to do earlier (only difference is this time it will open it in non-preview[Permanent] mode).